app/cafe.py

```diff
@@ -0,0 +1,21 @@
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+import datetime
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> None:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("You not vaccinated!")
+        elif visitor["vaccine"]["expiration_date"] < datetime.date.today():
+            raise OutdatedVaccineError("Your vaccine is lost date!")
+        elif visitor["wearing_a_mask"] is False:
+            raise NotWearingMaskError("Please, buy mask!")
```

Comment:
The error message 'Please, buy mask!' could be more descriptive. Consider changing it to 'You must wear a mask to enter the cafe.' to clearly state the requirement.